support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -70,17 +85,17 @@ The following is a sample guide for use with CloudFlare **only**; please note, t
     * Add a Setting -> Forwarding URL (you may need to specify a 301 redirect option)
     * Destination URL is https://<CNAME_VALUE>/$1. Replace <CNAME_VALUE> with the value displayed in the tracking domains section of the SparkPost UI. E.g.: for SparkPost US, this would be `spgo.io`; for SparkPost EU, this would be `eu.spgo.io`; for PMTA+Signals, refer to your user guide.
     * Save and Deploy (turn page rule on)
-    
+
     ![](media/enabling-https-engagement-tracking-on-sparkpost/SSL_full.png)
-    
+
 5. Cloudflare has Universal SSL for all accounts, but it's good to ensure that setting on the page rule is "SSL". This is required for how CloudFlare will validate the certificate on the origin.
 
     ![](media/enabling-https-engagement-tracking-on-sparkpost/page_rule.png)
 
-    
+
     More information on SSL options for Cloudflare can be found [here](https://support.cloudflare.com/hc/en-us/articles/200170416).
 
-6. Turn the page rule ON. 
+6. Turn the page rule ON.
 
 7. If you with to also set up [Mobile Universal and App Links](https://www.sparkpost.com/docs/tech-resources/deep-links-self-serve/), an additional page rule is necessary. A prerequisite for this configuration is that the desired universal link files (apple-app-site-assocation/assetlinks.json) are already hosted either via a CDN or webserver.
```

Comment:
Do we want this as a separate section so it isn't under "Create a domain" now that you have sections specifically outlined above on line 31

Reply:
We don't have this for hte other CDNs

Reply:
Agreed, I plan to rework the CloudFront sections and move this into the "deep links" document, I will open a separate PR for that.